src/python.cc

```diff
@@ -450,6 +394,12 @@ ModelInstanceState::GetInputTensor(
       in, &input_name, &input_dtype, &input_shape, &input_dims_count,
       &input_byte_size, &input_buffer_count));
 
+  // We need to create a new collector for every request because Python backend
+  // doesn't batch requests.
```

Comment:
What does this mean? The backend can receive a batch of requests, but are you saying that we don't actually collect each tensor into a contiguous buffer because that is not how they are delivered to the python code?

Reply:
I meant in the other backends like onnxruntime, a single buffer is created that contains all the requests and they do not call `ProcessTensor` for each request. But here we have to call it for every request because we do not batch multiple requests into a single request.

Reply:
So say "We need to create a new collector for every request because python backend sends each request individually to the python model".